### PR TITLE
DCOS 129 - microscaling demo tutorial

### PR DESCRIPTION
**Pull request by lizrice**
Adding a tutorial for running a Microscaling demo on DC/OS

### CODE REVIEW

File: 1.7/usage/tutorials/autoscaling/microscaling-queue.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+post_title: Microscaling based on queue length
+nav_title: Microscaling
+---
+
+[Microscaling](http://microscaling.org) adjusts the balance of tasks running within your compute cluster.
```
Comment (pyronicide) [inline 1.7/usage/tutorials/autoscaling/microscaling-queue.md]: Please move the links down to the bottom of the doc, it makes maintaining the docs much easier. (See the google card here: https://www.google.com/search?q=markdown%20reference%20links&rct=j).

File: 1.7/usage/tutorials/autoscaling/microscaling-queue.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+* Skip through steps 2 & 3 to use the default values. 
+* Navigate to the step 4 (Run) page and find your user ID and the default value for the queue we'll be using in the demo. You will use these as the values for environment variables MSS_USER_ID and AZURE_STORAGE_QUEUE_NAME later.
+
+![microscaling-step-4.png](../img/microscaling-step-4.png)
+
+# Get the microscaling scripts
+
+We have prepared some scripts to configure and start the four apps in Marathon. Go to a terminal on your local machine and get these scripts with the following command.
+
+```
```
Comment (pyronicide) [inline 1.7/usage/tutorials/autoscaling/microscaling-queue.md]: please add `bash` here for syntax highlighting.

File: 1.7/usage/tutorials/autoscaling/microscaling-queue.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+git clone http://github.com/microscaling/queue-demo
```
Comment (pyronicide) [inline 1.7/usage/tutorials/autoscaling/microscaling-queue.md]: Needs a `$` in the front of the line to show CLI input.

**Comment by pyronicide** [comment]
Looks pretty good. @joel-hamill mind giving it a pass?

**Comment by mhausenblas** [comment]
Yo! @pyronicide beat me to it. LGTM! Once above things are addressed, handing over to @joel-hamill for final review and publishing …

**Comment by lizrice** [comment]
Thanks for looking so fast @pyronicide @mhausenblas! Hope I've got those format changes all sorted now, shout if I missed anything.

**Comment by mhausenblas** [comment]
Perfect, thanks a lot @lizrice! @joel-hamill can you get it over the finish line pls?

**Comment by joel-hamill** [comment]
@mhausenblas yep, will do.

**Comment by joel-hamill** [comment]
@mhausenblas @lizrice I have added my edits here: https://github.com/dcos/dcos-docs/pull/346.

**Pull request merged**